tidy3d/components/geometry/utils.py

```diff
@@ -2,12 +2,18 @@
 
 from __future__ import annotations
 
+from enum import Enum
 from math import isclose
 from typing import Tuple, Union
 
 import numpy as np
+import pydantic as pydantic
```

Comment:
It should be equivalent to just 'import pydantic' ?

Reply:
The difference is that to access symbols from pydantic you have to use `pydantic.symbol` instead of just `symbol`. Same idea as the numpy import above, but just using the entire word.

Reply:
simpler with `import pydantic as pd` as in other places of tidy3d?